app/src/main/java/com/amaze/filemanager/utils/ServiceWatcherUtil.java

```diff
@@ -190,4 +225,22 @@ public void run() {
 
         handler.postDelayed(runnable, 0);
     }
-}
+
+    public interface ServiceWatcherInteractionInterface {
+
+        int STATE_HALTED = 0;
```

Comment:
Naming conventions, this variable in not final.

Reply:
Interface variables are final by default

Reply:
Ups, didn't know that.